pkg/processor/sourceprocessor/source_category_filler_test.go

```diff
@@ -53,14 +53,15 @@ func TestFillWithAnnotations(t *testing.T) {
 	attrs := pcommon.NewMap()
 	attrs.PutStr("k8s.namespace.name", "ns-1")
 	attrs.PutStr("k8s.pod.uid", "123asd")
+	attrs.PutStr("k8s.pod.name", "ABC")
 	attrs.PutStr("k8s.pod.annotation.sumologic.com/sourceCategory", "sc-from-annot-%{k8s.namespace.name}-%{k8s.pod.uid}")
-	attrs.PutStr("k8s.pod.annotation.sumologic.com/sourceCategoryPrefix", "%{k8s.pod.uid}-Prefix:")
+	attrs.PutStr("k8s.pod.annotation.sumologic.com/sourceCategoryPrefix", "%{k8s.pod.name}-%{k8s.pod.uid}-Prefix:")
```

Comment:
Why is prefix templating tested only with pod annotations? What about `TestFill` and `TestFillWithContainerAnnotations`?

Reply:
While this shouldn't matter, it's a good point that it should have more tests. Fixed.